src/module/item/SR5ItemSheet.ts

```diff
@@ -406,6 +408,30 @@ export class SR5ItemSheet extends ItemSheet {
         this.item.openSource();
     }
 
+    //Swap slots (att1, att2, etc.) for ASDF matrix attributes
+    async _onMatrixAttributeSelected(event) {
```

Comment:
This logic could potentially be commonized with the similar logic in SR5BaseActorSheet.ts:1732 that handles shuffling matrix attributes in the Matrix tab for Actors.

Reply:
Created an issue for it: https://github.com/smilligan93/SR5-FoundryVTT/issues/844 :)